MODULE.bazel

```diff
@@ -9,7 +9,7 @@ module(
 
 # Lower-bounds (minimum) versions for direct runtime dependencies.
 # Do not bump these unless rules_js requires a newer version to function.
-bazel_dep(name = "aspect_bazel_lib", version = "2.8.1")
+bazel_dep(name = "aspect_bazel_lib", version = "2.9.2")
```

Comment:
This is required for the bazelrc changes

Reply:
cool, can merge that as a pre-factor if it helps get this green and landed